io.nanoboyadvance.NanoBoyAdvance.metainfo.xml

```diff
@@ -0,0 +1,168 @@
+        <ul>
+          <li>UI: implement a 'Use integer scaling' option</li>
+          <li>UI: allow limiting the screen scale</li>
+          <li>UI: allow accessing the menu during fullscreen gameplay</li>
+          <li>UI: allow setting a custom save folder</li>
+          <li>Core: implement save compatibility with mGBA 0.10.0+</li>
+          <li>Core: implement basic support for the mGBA logging interface</li>
+          <li>PPU: rewrite the PPU to be mostly cycle-accurate</li>
+          <li>PPU: implement the GREENSWAP register</li>
+          <li>PPU: use the 6-th green channel bit during blending</li>
+          <li>PPU: round the blending result to the nearest integer</li>
+          <li>ARM: fix a minor timing oversight in ARM mode</li>
+          <li>ARM: SWP and SWPB should lock the bus (no DMA interleave is possible)</li>
+          <li>ARM: do not force-align mis-aligned PC in ARM mode</li>
+          <li>Bus: allow the CPU to execute idle cycles in parallel to DMA</li>
+          <li>Bus: more accurately emulate disabling the prefetch buffer</li>
+          <li>Bus: force the first CPU access after a DMA to be non-sequential</li>
+          <li>Bus: implement penalty for ROM code access during the last ROM prefetch cycle</li>
+          <li>IRQ: delay IO writes by one cycle</li>
+          <li>IRQ: delay update of the IE&amp;IF condition for unhalting the CPU</li>
+          <li>SIO: implement basic serial transfer timing</li>
+          <li>APU: emulate the master enable bit</li>
+          <li>APU: cancel a potentially pending event whan starting a channel</li>
+          <li>Scheduler: allow for (de)serialization of events for save states</li>
+        </ul>
+      </description>
+    </release>
+    <release version="1.6" date="2022-08-13" type="stable">
+      <url>https://github.com/nba-emu/NanoBoyAdvance/releases/tag/v1.6</url>
+    </release>
+    <release version="1.5" date="2022-05-29" type="stable">
+      <url>https://github.com/nba-emu/NanoBoyAdvance/releases/tag/v1.5</url>
+    </release>
+    <release version="1.4" date="2021-12-20" type="stable">
+      <url>https://github.com/nba-emu/NanoBoyAdvance/releases/tag/v1.4</url>
+    </release>
+    <release version="1.3" date="2021-02-01" type="stable">
+      <url>https://github.com/nba-emu/NanoBoyAdvance/releases/tag/1.3</url>
+    </release>
+    <release version="1.2" date="2020-07-30" type="stable">
+      <url>https://github.com/nba-emu/NanoBoyAdvance/releases/tag/v1.2</url>
+    </release>
+    <release version="1.1" date="2020-05-10" type="stable">
+      <url>https://github.com/nba-emu/NanoBoyAdvance/releases/tag/v1.1</url>
+    </release>
+    <release version="1.0" date="2020-05-10" type="stable">
+      <url>https://github.com/nba-emu/NanoBoyAdvance/releases/tag/v1.0</url>
+    </release>
+  </releases>
+  <url type="homepage">https://github.com/nba-emu/NanoBoyAdvance</url>
+  <url type="bugtracker">https://github.com/nba-emu/NanoBoyAdvance</url>
```

Comment:
why not a URL to the issue reporter instead?

Reply:
Just changed it to https://github.com/nba-emu/NanoBoyAdvance/issues